### PR TITLE
Feature/86cuvrpyf home page

### PR DESCRIPTION
**Pull request by tophatpatrick**
Pull request for home page

### CODE REVIEW

File: behind-the-veil-siteroot/imports/ui/components/pages/home/HomePage.jsx [hunk 1]
```diff
@@ -1,22 +1,201 @@
...
+                <div className="hidden md:flex md:items-start md:justify-center md:w-1/6">
+                    <Button onClick={goToNextImage} className="bg-white">
+                        <ChevronRightIcon className="size-10 stroke-[4]" />
+                    </Button>
+                </div>
+            </div>
+            {/* Container for left and right buttons to sort through images when screen is smaller than medium size*/}
+            <div className="inline-flex flex-row items-center justify-center md:hidden">
+                <div className="flex items-start justify-center w-[45%]">
+                    <Button onClick={goToPreviousImage} className="bg-white">
```
Comment (joshualoongwy) [inline behind-the-veil-siteroot/imports/ui/components/pages/home/HomePage.jsx:121]: small nitpick, are these buttons supposed to have transparent background color? bg-white makes them stand out a bit from the background 
image

**Review by joshualoongwy — COMMENTED** [review]
just a question about the arrow button bg colour, otherwise looks ready to merge

File: behind-the-veil-siteroot/imports/ui/components/pages/home/HomePage.jsx (hunk 1 above)
Comment (tophatpatrick) [reply behind-the-veil-siteroot/imports/ui/components/pages/home/HomePage.jsx:121]: Yeah, I can fix it though

Comment (tophatpatrick) [reply behind-the-veil-siteroot/imports/ui/components/pages/home/HomePage.jsx]: Fixed to make them transparent

**Review by joshualoongwy — APPROVED** [review]

**Pull request merged**